semgrep/semgrep/formatter/sarif.py

```diff
@@ -42,6 +44,42 @@ def _rule_match_to_sarif(rule_match: RuleMatch) -> Mapping[str, Any]:
             rule_match_sarif["suppressions"] = [{"kind": "inSource"}]
         return rule_match_sarif
 
+    @staticmethod
+    def _rule_match_to_sarif_fix(rule_match: RuleMatch) -> Optional[Mapping[str, Any]]:
+
+        # if rule_match.extra.get("dependency_matches"):
+        fixed_lines = rule_match.extra.get("fixed_lines")
+
+        description = rule_match.fix
+        if rule_match.fix_regex:
+            fix_regex = rule_match.fix_regex
+            description = (
+                f"s/{fix_regex.regex}/{fix_regex.replacement}/{fix_regex.count or 'g'}"
```

Comment:
count here is not actually how many that need to be replaced but is used internally for rule_match uniqueness when deduplicating rule_matches. Maybe just make the description generic for now? `Semgrep rule suggested fix`

Reply:
Got it